addons/l10n_ch/views/account_invoice_view.xml

```diff
@@ -47,6 +47,7 @@
                         attrs="{'invisible':['|', ('state', '!=', 'posted'),
                                              '|', ('l10n_ch_isr_sent', '=', True),
                                              '|', ('move_type', '!=', 'out_invoice'),
+                                             '|', ('partner_bank_id', '=', False),
```

Comment:
not really needed, because there is already an error message. Maybe improve the error message. Modifying backend xml won't apply on existing DB automatically, because it is copied in the customer DB, so it is not updated by itself when modified in the code